docker-compose.yml

```diff
@@ -6,25 +6,41 @@ services:
     image: postgres:11-alpine
     # To preserve data between runs of docker-compose, we mount a folder from the host machine.
     volumes:
-     - dbdata:/var/lib/postgresql/data
+      - dbdata:/var/lib/postgresql/data
     environment:
-    - POSTGRES_PASSWORD=developmentpassword
+      - POSTGRES_PASSWORD=developmentpassword
+
+  redis:
+    image: redis
 
   web:
     build:
       context: .
       cache_from:
-        - $DOCKER_REPO:${BRANCH_TAG:-latest}
-    image: $DOCKER_REPO:${BRANCH_TAG:-latest}
-    command: ash -c "rm -f tmp/pids/server.pid && bundle exec rails server -p 3001 -b '0.0.0.0'"
+        - dfedigital/register-trainee-teacher-data:${BRANCH_TAG:-latest}
+    image: dfedigital/register-trainee-teacher-data:${BRANCH_TAG:-latest}
+    command: ash -c "bundle exec rails db:create && bundle exec rails db:migrate && bundle exec rails server -p 3001 -b '0.0.0.0'"
```

Comment:
@misaka can you confirm it makes sense? that's what @felixtheflex and I came up with to make it work automatically

Reply:
Ya, I think so. The `rails db:create` seems warranted because it's `docker-compose.yml` ... we don't rely on that in production or in any deployed envs, so that should be ok.

Reply:
cheer...many thanks for clarification.